Unity android build obsolete warning (#393)
Microsoft/AppCenter-SDK-Unity: CHANGELOG.md

## Version 2.6.1 (Under development)

### App Center

#### Android

> * **[Fix]** Fix the Unity warning that appears during the building app for Android.

Reviewer: ```suggestion
* **[Fix]** Fix a Unity warning that appears when building for Android.
```
